app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
```

Comment:
- [ ] Kindly indicate the relationship with the posts model (table), since it's using the author id and not user id.

`has_many :posts, foreign_key: 'author_id'`